tracer/test/Datadog.Trace.TestHelpers.AutoInstrumentation/InstrumentationVerification.cs

```diff
@@ -18,26 +18,30 @@ internal static class InstrumentationVerification
 {
     /// <summary>
     /// Configuration key for enabling or disabling the instrumentation verification.
-    /// Default is value is disabled.
+    /// Default is value is disabled in tests (should be enabled in production).
```

Comment:
```suggestion
    /// Default value is enabled
```

Is the original text here accurate? If it is, we should change it, I don't see any justification for behavior in tests to be different than behavior in production here?

Reply:
Fixed in d6e34e581c70cc2271806c12db2c3d4f2caf78f1